.github/workflows/vulnerability-scanner-generate-database.yml

```diff
@@ -62,6 +64,16 @@ jobs:
           fi
           echo "Git branch: $(git branch | grep "*")"
 
+          cp -r /tmp/{actions,workflows} .github/
+
+          template=src/wazuh_modules/vulnerability_scanner/indexer/template/index-template.json
+          if [ -f "$template" ]; then
+            if [ ! -d "templates" ]; then
+              mkdir templates
+            fi
+            cp "$template" templates/vd_states_template.json
+          fi
```

Comment:
This should be done in another step

Reply:
Add a comment explaining this

Reply:
Added some comments, although is not too much complex to the already existent logic there.